internal/pkg/agent/cmd/otel.go

```diff
@@ -118,3 +123,22 @@ func runCollector(cmdCtx context.Context, configFiles []string) error {
 	return goerrors.Join(errs...)
 
 }
+
+func prepareEnv() error {
+	if _, ok := os.LookupEnv("STORAGE_DIR"); !ok {
```

Comment:
Can you use STATE_PATH here as well?

What I am trying to do is avoid introducing a new environment variable we may decide to remove in the future, trying to keep our public API as small as possible while we figure out how everything fits together. 

STATE_PATH isn't going anywhere without making it a breaking change for many people, so it doesn't increase the surface area of configuration we need to maintain.